oxidation/parsec_cli/parsec_sequestre_dump/src/model.rs

```diff
@@ -0,0 +1,47 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use parsec_api_crypto::SecretKey;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Queryable, Serialize, Deserialize, PartialEq)]
+pub struct Block {
+    pub block_id: Vec<u8>,
+    pub data: Vec<u8>,
+    pub author: i32,
+    pub size: i32,
+    pub created_on: f32,
+}
+
+#[derive(Debug, Queryable, Serialize, Deserialize, PartialEq)]
+pub struct VlobAtom {
+    pub vlob_id: Vec<u8>,
+    pub version: i32,
+    pub blob: Vec<u8>,
+    pub size: i32,
+    pub author: i32,
+    pub timestamp: f32,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub struct Data {
+    pub blocks: Vec<Block>,
+    pub vlob_atoms: Vec<VlobAtom>,
+    pub role_certificates: Vec<Vec<u8>>,
+    pub user_certificates: Vec<Vec<u8>>,
+    pub device_certificates: Vec<Vec<u8>>,
+}
+
+impl Data {
+    pub fn save(&self, key: &SecretKey, path: &str) {
```

Comment:
Ha ! I just realized `save` here means save *to disk* and not save *to database*

Anyway, we should use the `parsec_api_types` schema to do the deserialization

(don't do the consistency checking on expected timestamp & expected author for now, this is not vital and we should first think a bit about the architecture & trustchani public api before doing this)

Reply:
Aaaand one last thing: I think it would be better to keep a separation between the data fetched from sqlite (i.e. those structs) and the work we want to do on them (e.i. this method)

Hence you should have a separate file with functions (and not methods) for the save on disk logic

Reply:
I would prefer a name like `dump`, That brings us back to the idea of dumping the database on the disk